### PR TITLE
release: v4.8.3

### PR DESCRIPTION
**Pull request by jsjoeio**
New release for 4.8.3

### CODE REVIEW

File: ci/helm-chart/Chart.yaml [hunk 1]
```diff
@@ -20,4 +20,4 @@ version: 3.3.2
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 4.8.2
+appVersion: 4.8.3
```
Comment (code-asher) [inline ci/helm-chart/Chart.yaml:23]: Chart version needs updating as well

Comment (jsjoeio) [reply ci/helm-chart/Chart.yaml:23]: Nice catch! Updating now

**Review by code-asher — APPROVED** [review]

**Review by code-asher — APPROVED** [review]
Nice catch on the install script change, I completely forgot about it despite approving it this morning.  :facepalm:

**Comment by jsjoeio** [comment]
Lol no worries! At least I could publish a release without waiting to make that change in the source code!

**Pull request merged**